lib/dotcom_web/components/components.ex

```diff
@@ -261,9 +262,24 @@ defmodule DotcomWeb.Components do
 
   def promo_banner(assigns) do
     ~H"""
-    <a href={@href} class={"block bg-amethyst-90 text-black text-center #{@class}"} target="_blank">
+    <a href={@href} class={"block bg-amethyst-80 text-black text-center #{@class}"} target="_blank">
       {render_slot(@inner_block)}
     </a>
     """
   end
+
+  def cr_survey_promo_banner(assigns) do
```

Comment:
Style nit: I would put this component into a separate module, so that when it's time to remove it, it's a cleaner excision.

Reply:
eh, _would_ it be cleaner? you'd have to remove the new module and its references...

Reply:
My thought was that then you can just delete the file and clean up the compiler errors and warnings.

But I suppose that's true of having the component in this file as well.